components/FeedItemCard.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <v-hover>
+    <template v-slot="{ isHovering, props }">
+      <v-card v-bind="props">
+        <v-img :src="src()" aspect-ratio="1">
```

Comment:
see below comment about making `src` a computed value.  after you do, you'll just have `:src="src"` without the parens